.github/workflows/package.yml

```diff
@@ -0,0 +1,54 @@
+name: package
+
+on:
+  push:
+    branches:
+      - 'v[0-9]+.*'
```

Comment:
nightly package should based on master branch.

Reply:
OK